src/main/app/src/app/shared/model/indicatie.ts

```diff
@@ -5,12 +5,24 @@
 
 export class Indicatie {
     naam: string;
+    icon: string;
+    outlined: boolean;
     primary: boolean;
     toelichting: string;
 
-    constructor(naam: string, primary: boolean, toelichting: string) {
+    constructor(naam: string, icon: string, toelichting: string) {
         this.naam = naam;
-        this.primary = primary;
+        this.icon = icon;
         this.toelichting = toelichting;
     }
+
+    temporary(): Indicatie {
+        this.primary = true;
+        return this;
+    }
+
+    alternate(): Indicatie {
```

Comment:
kan je deze hernoemen naar iconOutlined dan is het duidelijk wat deze doet

Reply:
Moah... dan moet die andere ook colourPrimary gaan heten. Ik wilde we eigenlijk een klein beetje extra info aan toevoegen. Maar ik geef toe dat alternate niet heeeeeel informatief is. ;-)

Ik laat het even zo.